django_project/lesson/models/further_reading.py

```diff
@@ -0,0 +1,37 @@
+# coding=utf-8
+"""Further reading model definitions for lesson apps.
+
+"""
+import logging
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from lesson.models.worksheet import Worksheet
+
+logger = logging.getLogger(__name__)
+
+
+class FurtherReading(models.Model):
+    """Further reading lesson model."""
+
+    worksheet = models.ForeignKey(Worksheet)
+
+    text = models.CharField(
+        help_text=_('Text of the further reading.'),
+        max_length=200,
+    )
+
+    link = models.CharField(
+        help_text=_('Link of the further reading.'),
```

Comment:
Further reading link

Reply:
This can be null true blank true - better to explicitly declare every field as to whether it can be null and blank.....

Reply:
Btw, few days ago I read that it's better not use `null=True` for CharField or TextField (unless we need it to be unique). https://docs.djangoproject.com/en/1.8/ref/models/fields/#null